atef/bin/check.py

```diff
@@ -312,7 +312,7 @@ def check_and_log(
     )
 
 
-def main(
+async def main(
```

Comment:
Intuitively this seems like a really good design decision to me as we add various tools that may take some time to do their checks in non-uniform ways. Does this open the possibility of running checks in parallel and recombining their results later?

Reply:
I think so, too.  Moving to asyncio should make running things in parallel easier among other things.